test/bye_bye_bye/utils_test.exs

```diff
@@ -263,4 +263,103 @@ defmodule ByeByeBye.UtilsTest do
       assert result == %{}
     end
   end
+
+  describe "protox_struct_to_map/1" do
+    test "converts a simple Protox struct to a map" do
```

Comment:
could you `doctest` the function here?

Reply:
I pushed up the change for this. I have mixed feelings about it. It seems doctest matches the whole data structure rather than pattern matching, so the doc got a lot more verbose.